docs/source/index.rst

```diff
@@ -9,40 +9,36 @@
 Introduction
 ============
 
-Cartopy is a Python package designed to make drawing maps for data analysis and visualisation as easy as possible.
+Cartopy is a Python package designed for geospatial data processing in order to produce maps and other geospatial data analyses.
 
-Cartopy makes use of the powerful PROJ.4, numpy and shapely libraries and has a simple and intuitive
-drawing interface to Matplotlib for creating publication quality maps.
+Cartopy makes use of the powerful PROJ.4, numpy and shapely libraries and includes an programatic interface
```

Comment:
"a programmatic"?

Reply:
A reasonable nit - it is on the homepage. 😄